beacon-chain/cache/tracked_validators.go

```diff
@@ -47,3 +47,9 @@ func (t *TrackedValidatorsCache) Validating() bool {
 	defer t.Unlock()
 	return len(t.trackedValidators) > 0
 }
+
+func (t *TrackedValidatorsCache) Size() int {
+	t.Lock()
+	defer t.Unlock()
+	return len(t.trackedValidators)
+}
```

Comment:
Could it be beneficial to add a note directly to the code that the accuracy of this is not 100%?

Reply:
this does return the size of the cache, it's more that the metric might not be fully representative